tests/system/python/conftest.py

```diff
@@ -668,6 +670,29 @@ def _disable_sch(fledge_url, sch_name):
     return _disable_sch
 
 
+@pytest.fixture(scope="function", autouse=True)
+def collect_support_bundle(request, fledge_url):
+    def _collect_support_bundle():
+        PROJECT_ROOT = Path(__file__).absolute().parent.parent.parent.parent
+        
+        try:
+            jdoc = utils.post_request(fledge_url, "/fledge/support", None)
+            assert jdoc["bundle created"]
+            copy_to_cmd = "mkdir -p {0}/support/ && cp -r {1} {0}/support".format(PROJECT_ROOT, jdoc['bundle created'])
+            subprocess.run([copy_to_cmd], shell=True, check=True)
+        except Exception as e:
+            print("\n Failed to get Support Bundle. \n {}".format(str(e)))
+            print("Copying syslog")
+            log_file = "/var/log/syslog"
+            if pytest.PKG_MGR == 'yum':
+                log_file = "/var/log/messages"  
+            ts = time.strftime("%Y_%m_%d_%H_%M_%S")
+            copy_to_cmd = "mkdir -p {0}/support/logs/ && cp {1} {0}/support/logs/syslog_{2}".format(PROJECT_ROOT, log_file, ts)
```

Comment:
IN case of exception we should only get the Fledge related logs. Not the full dump from path.

Reply:
ok, I'll update the code to only get logs related to Fledge.